modin/pandas/test/test_io.py

```diff
@@ -66,6 +66,7 @@
     default_to_pandas_ignore_string,
     parse_dates_values_by_id,
     time_parsing_csv_path,
+    test_data as utils_test_data,
```

Comment:
```suggestion
    test_data,
```
Why can't we use this as is?

Reply:
This is probably to avoid ambiguity - the file also defines a `TEST_DATA` (in all upper case), so while there is no namespace shadowing, its probably better for code comprehension + search ability to not have two different variables with the same name barring case.